.github/CODEOWNERS

```diff
@@ -36,10 +36,8 @@ sycl/doc/extensions/ @intel/dpcpp-specification-reviewers
 
 # Unified Runtime
 unified-runtime/ @intel/unified-runtime-reviewers
-# TODO: Use specific UR Level Zero adapter team
-unified-runtime/*/adapters/level_zero/ @intel/unified-runtime-reviewers
-# TODO: Use specific UR OpenCL adapter team
-unified-runtime/*/adapters/opencl/ @intel/unified-runtime-reviewers
+unified-runtime/*/adapters/level_zero/ @intel/unified-runtime-reviewers-level-zero
+unified-runtime/*/adapters/opencl/ @intel/unified-runtime-reviewers-opencl
```

Comment:
Is it okay to go ahead with the creation of these new code owner teams?

Reply:
imo sure

Reply:
These have now been created so I'll go ahead with merging this.